### DISCUSSION TITLE
error when encrypting - what does this mean?

### DISCUSSION THREAD

**Discussion by tinker2much** [gitbls/sdm#167] [General]
I'm trying to encrypt an sd card in a pi4, bookworm, updated, using the latest sdm. 
...
3568179+0 records in
3568179+0 records out
> 1970-01-01 00:20:30 Enable luks2 encryption on '/dev/mcb1k0p2'
WARNING: Device /dev/mmeblk0p2 already contains a 'ext4' superblock signature.
WARNING!
This will overwrite data on /dev/mmcblk0p2 irrevocably.
Are you sure? (Type 'yes' in capital letters): YES
Operation aborted.
WARNING: Device /dev/mmcbIk0p2 already contains a 'ext4' superblock signature.

**Comment by gitbls** [comment]
Well, that's very interesting. Investigating.

**Comment by gitbls** [comment]
Was there anything after the line with WARNING? The only thing that looks curious to me is the "Operation aborted" message, but what happened after that?

**Comment by tinker2much** [comment]
I hadn't tried to respond last night, just left the machine sitting there. I did so this morning, and it repeats, but then it takes my response and proceeds:
![IMG_8385](https://github.com/gitbls/sdm/assets/57112204/c9cfe3fe-0b3c-4f78-be2b-d3d730c16b83)

**Comment by tinker2much** [comment]
... and it finishes. seems ok

**Comment by gitbls** [comment]
I took a look at the cryptsetup sources (https://gitlab.com/cryptsetup/cryptsetup). There are about a dozen places where it prints "Operation aborted". I found a couple of likely candidates from that, but no clear hint as to what might have caused it. 

File this under "interesting", and keep until something that's reproducible pops up, I guess?
